.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Describe your changes
+
+
+## Type of change
+
+Please delete options that are not relevant (and this line).
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
```

Comment:
I don't understand this particular kind of change. shouldn't the documentation update be part of the PR

Reply:
I'll remove that option.